app/views/partials/_recovery_code.html.slim

```diff
@@ -8,7 +8,8 @@
   .navy.monospace
     - code.split(' ').each do |word|
       p.bold.mb0.h3
-        = word
+        span.recovery-code-word
```

Comment:
@pkarman In the modal, I'm passing the code into a `js.erb` file. If I was using selectors I would probably make a data attribute, `data-key-word` or something along those lines.